spec/derivative_services/audio_derivative_service_spec.rb

```diff
@@ -4,7 +4,7 @@
 include ActionDispatch::TestProcess
 
 RSpec.describe AudioDerivativeService do
-  it_behaves_like "a Valkyrie::Derivatives::DerivativeService"
+  # it_behaves_like "a Valkyrie::Derivatives::DerivativeService"
```

Comment:
Do we want to change the shared spec? Alternatively, we could keep the signature the same, but use the change_set's ID to reload the object instead of passing the ID...

Reply:
I think we'll want to change the shared spec at some point.

Reply:
👍 I created a ticket for tracking whether we want to change the API: https://github.com/samvera-labs/valkyrie-derivatives/issues/3

Meanwhile, do you want to mark the shared spec pending?

Reply:
:+1: